Apromore-Core-Components/Apromore-Portal/src/main/webapp/themes/ap/share/js/index.js

```diff
@@ -1,5 +1,11 @@
 Ap.share = Ap.share || {};
 
+Ap.share.revertCombobox = (rowGuid, access) => {
+  let commbobox = zk.Widget.$(jq("[data-id='" + rowGuid + "']")[0]);
+  commbobox.setValue(access);
+  zAu.send(new zk.Event(commbobox, 'onChange', access)); // sync with server
```

Comment:
Is "commbobox" deliberately spelled with two "m"s?

Reply:
Oops. Fixed now, even though it could be interpreted as communication box ;).